query-engine/src/chat_interface.py

```
@@ -0,0 +1,98 @@
+import asyncio
+from indexer import CSVIndexer
+import sys
+from rich.console import Console
+from rich.panel import Panel
+from rich.markdown import Markdown
```

Ruff (0.8.2)

`rich.markdown.Markdown` imported but unused

Remove unused import: `rich.markdown.Markdown`

(F401)